crates/ln-dlc-node/src/ldk_node_wallet.rs

```diff
@@ -50,6 +56,8 @@ where
     inner: Mutex<bdk::Wallet<D>>,
     settings: RwLock<WalletSettings>,
     fee_rate_estimator: Arc<F>,
+    // Only cleared upon restart. This means that if a locked outpoint ends up unspent, it will
+    // remain locked until the binary is restarted.
     locked_outpoints: Mutex<Vec<OutPoint>>,
```

Comment:
Unrelated to this PR, but maybe these `locked_outpoints` should be attached with a timestamp with which we can reuse locked outputs if they haven't been used e.g. more than 30 mins?